crm_claim_code/models/crm_claim.py

```diff
@@ -29,3 +30,14 @@ def copy(self, default=None):
             default = {}
         default['code'] = self.env['ir.sequence'].get('crm.claim')
         return super(CrmClaim, self).copy(default)
+
+    @api.multi
+    @api.depends('code')
```

Comment:
I'm not saying it's 100% needed, just that this is how the new API works and the decorator uses it exactly as intended (and unlike the v7/v8 compatibility layer which actually slows things down, this will not have negative perf. impact, the opposite if I understand what the next steps in this area will be).